microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -281,18 +289,115 @@ DevTools now tracks all of the objects that were GC'd during the recording.  Use
 
 If you are investigating objects that were only GC'd during specific major or minor GC operations, configure the settings appropriately to track the operation you care about. To learn more about the differences between major and minor GC, see [Trash talk: the Orinoco garbage collector | V8 JavaScript engine developer blog](https://v8.dev/blog/trash-talk).
 
+
+<!-- ====================================================================== -->
+## Find DOM tree memory leaks from detached elements
+
+A DOM node is only garbage-collected when there are no references to the node from either the DOM tree or JavaScript code running on the page.  A node is said to be "detached" when it is removed from the DOM tree but some JavaScript still references it.  Detached DOM nodes are a common cause of memory leaks.
+
+
+<!-- ------------------------------ -->
+#### Tools for investigating detached elements
+
+| Tool | Pros | Cons | Docs |
+|---|---|---|---|
+| **Memory** tool > **Detached elements** option button (profiling type) | Shows elements as DOM tree nodes. | Doesn't link to JavaScript source code. | [Debug DOM memory leaks ("Detached elements" profiling type)](./dom-leaks-memory-tool-detached-elements.md) |
+| **Memory** tool > **Heap snapshot** option button (profiling type) > **Detached** | The detached element has a link to its JavaScript source code.<!-- todo: how get to JS? --> | Shows elements as objects in memory. | [Record heap snapshots using the Memory tool ("Heap snapshot" profiling type)](./heap-snapshots.md) |
+| **Detached Elements** tool | Shows elements as DOM tree nodes.  The detached element has a link to its JavaScript source code. | Deprecated. | [Debug DOM memory leaks by using the Detached Elements tool](./dom-leaks.md) |
```

Comment:
Instead of Pros and Cons, I would phrase this as use cases.

```suggestion
| Tool | Use case | Docs |
|---|---|---|---|
| **Memory** tool > **Detached elements** option button (profiling type) | Shows detached elements only, as DOM tree nodes.  | [Debug DOM memory leaks ("Detached elements" profiling type)](./dom-leaks-memory-tool-detached-elements.md) |
| **Memory** tool > **Heap snapshot** option button (profiling type) > **Detached** | Shows all objects in memory, including detached elements, which requires filtering, with links to the JavaScript source code in the **Retainers** panel. | [Record heap snapshots using the Memory tool ("Heap snapshot" profiling type)](./heap-snapshots.md) |
| **Detached Elements** tool | Shows detached elements only, as DOM tree nodes, with links to the JavaScript source code. | Deprecated. | [Debug DOM memory leaks by using the Detached Elements tool](./dom-leaks.md) |
```

Reply:
result:
```
| Tool | Use case | Docs |
|---|---|---|
| **Memory** tool > **Detached elements** option button (profiling type) | Shows detached elements only, as DOM tree nodes.  | [Debug DOM memory leaks ("Detached elements" profiling type)](./dom-leaks-memory-tool-detached-elements.md) |
| **Memory** tool > **Heap snapshot** option button (profiling type) > **Detached** | Shows all objects in memory, including detached elements, which requires filtering, with links to the JavaScript source code in the **Retainers** panel. | [Find DOM tree memory leaks ("Heap snapshot" profiling type > Detached)](./heap-snapshots.md#find-dom-tree-memory-leaks-heap-snapshot-profiling-type--detached) in _Record heap snapshots using the Memory tool ("Heap snapshot" profiling type)_. |
| **Detached Elements** tool | Shows detached elements only, as DOM tree nodes, with links to the JavaScript source code. | [Debug DOM memory leaks by using the Detached Elements tool](./dom-leaks.md) |
```
rendered preview:
![image](https://github.com/user-attachments/assets/c2d76bb9-12c5-4c0b-aaf9-cb507612dec8)